dist/stat_data.py

```diff
@@ -60,6 +60,10 @@ class CapacityStat(Stat):
     prefix = 'capacity'
     def __init__(self, name, desc, flags=''):
         Stat.__init__(self, name, CapacityStat.prefix, desc, flags)
+class CheckpointStat(Stat):
+    prefix = 'checkpoint'
+    def __init__(self, name, desc, flags=''):
+        Stat.__init__(self, name, CheckpointStat.prefix, desc, flags)
```

Comment:
This should be alphabetized after CheckpointCleanup. I agree with @mm-ng  about considering only one checkpoint stat category, maybe not having a separate one for cleanup.

Reply:
I've merged these - I think you and Monica made the right call here.